sensu-plugins-java.gemspec

```diff
@@ -22,21 +22,23 @@ Gem::Specification.new do |s|
   s.platform               = Gem::Platform::RUBY
   s.post_install_message   = 'You can use the embedded Ruby by setting EMBEDDED_RUBY=true in /etc/default/sensu'
   s.require_paths          = ['lib']
-  s.required_ruby_version  = '>= 2.0.0'
+  s.required_ruby_version  = '>= 2.1.0'
   s.summary                = 'Sensu plugins for Java'
   s.test_files             = s.files.grep(%r{^(test|spec|features)/})
   s.version                = SensuPluginsJava::Version::VER_STRING
 
-  s.add_runtime_dependency 'sensu-plugin', '~> 1.2'
   s.add_runtime_dependency 'english', '0.6.3'
+  s.add_runtime_dependency 'rest-client', '~> 2.0'
```

Comment:
I missed this earlier but if we are now requiring `rest-client` it requires a c compiler to install this plugin this constitutes a breaking change and need to be called out like this:
```
### Breaking Changes
- dropped ruby 2.1 support (@yuri-zubov sponsored by Actility, https://www.actility.com)
- added `rest-client` as a dependency which requires you to have a local c compiler present to install this plugin (@yuri-zubov sponsored by Actility, https://www.actility.com)

￼```

Reply:
done